src/controllers/update/update.manage.trusts.tell.us.about.the.former.bo.controller.ts

```diff
@@ -82,9 +83,9 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
   }
 };
 
-const getPageProperties = (trust: Trust, formData?: TrustHistoricalBeneficialOwnerForm, errors?: FormattedValidationErrors) => ({
+const getPageProperties = (trust: Trust, formData: TrustHistoricalBeneficialOwnerForm, url: string, errors?: FormattedValidationErrors) => ({
   backLinkUrl: getBackLink(trust.review_status?.reviewed_former_bos),
-  templateName: UPDATE_MANAGE_TRUSTS_TELL_US_ABOUT_THE_FORMER_BO_PAGE,
+  templateName: url ? url.replace(`${UPDATE_LANDING_URL}/`, "") : UPDATE_MANAGE_TRUSTS_TELL_US_ABOUT_THE_FORMER_BO_PAGE,
```

Comment:
There is already a constant you can use in place of ${UPDATE_LANDING_URL}/ 

https://github.com/companieshouse/overseas-entities-web/blob/main/src/config/index.ts#L253